application/src/main/java/uk/nhs/digital/uec/api/config/LocalConfig.java

```diff
@@ -0,0 +1,29 @@
+package uk.nhs.digital.uec.api.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.web.authentication.preauth.AbstractPreAuthenticatedProcessingFilter;
+import uk.nhs.digital.uec.api.filter.LocalAccessTokenFilter;
+
+@Profile("local")
+@Configuration
+public class LocalConfig extends WebSecurityConfigurerAdapter {
```

Comment:
Don't think we need this class any more?

Reply:
We need this class as the filter now are different for local and rest of the profiles